book/getting-started/linux-setup.md

```diff
@@ -90,61 +51,26 @@ sudo update-alternatives --install /usr/bin/gcc gcc \
   /usr/bin/gcc-5 60 --slave /usr/bin/g++ g++ /usr/bin/g++-5
 ```
 
-### Install prce2
-
-#### Xenial Ubuntu:
-
-```bash
-sudo apt-get install -y libpcre2-dev
-```
-
-#### Trusty Ubuntu:
-
-*Note:* some older versions of Ubuntu do not supply a prce2
-package. If you get an error that no package exists (`E: Package
-'libpcre2-dev' has no installation candidate`) then you will need to
-install from source like this:
-
-```bash
-cd ~/
-wget ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/pcre2-10.21.tar.bz2
-tar xjvf pcre2-10.21.tar.bz2
-cd pcre2-10.21
-./configure --prefix=/usr
-make
-sudo make install
-```
-
 ### Installing ponyc
 
-Now you need to install the Wallaroo Labs fork of the Pony compiler `ponyc`. Run:
-
-```bash
-cd ~/
-wget https://github.com/WallarooLabs/ponyc/archive/wallaroolabs-19.2.14.tar.gz
-tar xzfv wallaroolabs-19.2.14.tar.gz
-cd ponyc-wallaroolabs-19.2.14
-sudo make config=release install
```

Comment:
While removing the custom compiling install instructions makes sense for folks to get up and running quickly, we'll likely want to retain these instructions in a `Advanced` or `Best Performance` scenario because compiling `ponyc` from source has a measurable impact on performance compared to relying on the CI built variant.

Reply:
I'm going to open an issue for this.  Our current goal is to get people up and running quickly.

Reply:
Opened #1602, plagiarizing freely